cmp/grid/GridModel.js

```diff
@@ -228,6 +241,7 @@ export class GridModel {
 
         contextMenu,
         useVirtualColumns = false,
+        autosizeOptions = {showMask: true},
```

Comment:
Don't think we want to default it like this -- need to merge in that default in the code, so changing another option does not step on this one